discord/install.sh

```diff
@@ -59,6 +59,13 @@ EOT
 	echo 'YoutubeDLPath = "/usr/local/bin/youtube-dl"' >> config.ini.configured
 	echo "Successfully installed youtube-dl"
 	;;
+"yt-dlp")
```

Comment:
The `yt-dlp` task should just replace the old `youtube-dl` one, I don't think there is a need to keep it around.